comptests/ha/test_ha_shutdown_cluster.py

```diff
@@ -275,54 +278,58 @@ def test_scale_pod_verify_ha_alerts(self):
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 4: Verified k8s resource monitor log for skipping alert message")
+
+        LOGGER.info("Step 5: Verify HealthMonitor should not send degraded alert to comp Hare ")
+        LOGGER.info('Take word count of health monitor logs after pod operation')
+        wc_after = HAK8SCompLib.get_word_count(node_obj, log_index=2)
+        assert_utils.assert_equal(wc_before, wc_after,
                                   "Failed:cluster degraded alert sent to Hare.")
-        LOGGER.info("Step 4:Verified HealthMonitor should not send degraded alert to comp Hare")
-        LOGGER.info("Step 5: Start pod by making replicas=1")
+        LOGGER.info("Step 5: Successfully verified HealthMonitor should not send degraded alert "
+                    "to comp Hare")
+
+        LOGGER.info("Step 6: Start pod by making replicas=1")
         resp = self.ha_obj.restore_pod(pod_obj=self.node_master_list[0],
                                        restore_method=self.restore_method,
                                        restore_params={"deployment_name": self.deployment_name})
         LOGGER.debug("Response: %s", resp)
         assert_utils.assert_true(resp[0], f"Failed to restore pod by {self.restore_method} way")
-        LOGGER.info("Step 5: Successfully started the pod again by making replicas=1")
+        LOGGER.info("Step 6: Successfully started the pod again by making replicas=1")
         self.restore_pod = False
-        LOGGER.info("Step 6:Verify HealthMonitor should not send degraded alert to comp Hare")
-        ha_wc_count_after_scale = node_obj.execute_cmd(ha_wc_count_cmd)
-        if isinstance(ha_wc_count_after_scale, bytes):
-            ha_wc_count_after_scale = str(ha_wc_count_after_scale, 'UTF-8')
-        LOGGER.info("cluster degraded alerts not seen %s", ha_wc_count_after_scale)
-        assert_utils.assert_equal(ha_wc_count, ha_wc_count_after_scale,
-                                  "Failed:cluster degraded alert sent to hare .")
-        LOGGER.info("Step 6:Verified HealthMonitor should not send degraded alert to comp Hare")
-        LOGGER.info("Step 7: Shutdown entire cortx Cluster")
+
+        LOGGER.info("Step 7: Verify k8s resource monitor for skipping alert message")
+        time.sleep(HA_CFG["common_params"]["20sec_delay"])
+        resp = HAK8SCompLib.check_string_in_log_file(node_obj, "cluster_stop_key",
+                                                     common_const.HA_SHUTDOWN_LOGS[1], lines=1)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 7: Verified k8s resource monitor log for skipping alert message")
+
+        LOGGER.info("Step 8:Verify HealthMonitor should not send degraded alert to comp Hare")
+        LOGGER.info('Take word count of health monitor logs after pod operation')
+        wc_after = HAK8SCompLib.get_word_count(node_obj, log_index=2)
```

Comment:
Update it to get_ha_log_wc.
Lib method name is changed, you need to change the name wherever it is called.

Reply:
Updated